docs/setup/install-existing.md

```diff
@@ -64,46 +64,58 @@ To install AAD Pod Identity to your cluster:
   kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/v1.5.5/deploy/infra/deployment.yaml
   ```
 
-Next we need to create an Azure identity and give it permissions ARM.
+Next we need to create an Azure identity and give it permissions to ARM. This identity will then be assigned to AGIC Pod using AAD Pod Identity.
 Use [Cloud Shell](https://shell.azure.com/) to run all of the following commands and create an identity:
 
-1. Create an Azure identity **in the same resource group as the AKS nodes**. Picking the correct resource group is
-important. The resource group required in the command below is *not* the one referenced on the AKS portal pane. This is
-the resource group of the `aks-agentpool` virtual machines. Typically that resource group starts with `MC_` and contains
- the name of your AKS. For instance: `MC_resourceGroup_aksABCD_westus`
+1. Create a User assigned identity in the same resource group as AKS cluster.
```

Comment:
Add a note when the UMI is not in the same resource group as AKS cluster?

Reply:
let me change the wording. these steps can work for any resource group.